### PR TITLE
Fix Mama Turtle Room

### PR DESCRIPTION
**Pull request by kjbranch**
- Fix strats by modeling Mama Turtle in sleeping and awake states.
- Add `canRiskPermanentLossOfAccess` to a missing `not` in Maridia

### CODE REVIEW

File: region/maridia/outer.json [hunk 1]
```diff
@@ -3644,7 +3667,13 @@
                       "SpaceJump"
                     ]}
                   ],
-                  "note": "To avoid getting hit, shoot one of her babies to wake her up, then quickly get on her back."
+                  "obstacles": [
```
Comment (osse101) [inline region/maridia/outer.json:3670]: Could you add `or:HiJump` to this strat

**Review by osse101 — COMMENTED** [review]
Its close to looking like we want a 2->3 in this room, but there's a strat or two that only visit 2 for the item and don't land on the grapple block.
3->2 would be possible too through the hijump+spacejump path but that's unnecessary.

**Comment by kjbranch** [comment]
All strats that go from 2->3 and land on the block are accounted for in 1->3. It's very few of them. Most strats from 1->2 don't land on the block. We could make this a little cleaner with a more major change, like adding a node or an abstract obstacle, but I don't think it's necessary at this time. I'll add a fixme for a suggestion to cleanup the room.

File: region/maridia/outer.json [hunk 2]
```diff
@@ -3538,6 +3546,11 @@
                   "requires": [
                     "h_canNavigateUnderwater",
                     "Grapple"
+                  ],
+                  "note": [
+                    "To get out of the water, ride Mama Turtle. To avoid getting hit, shoot one of her babies to wake her up, then quickly get on her back.",
+                    "If Mama Turtle is in her starting location, stand on the rightmost side of her to be aligned below the grapple block.",
+                    "With a fail, if you havent moved horizontally, simply fall straight down to land safely back on to her."
```
[the remark this answers is not in the document]
Comment (kjbranch) [reply region/maridia/outer.json]: I was debating making this even more clear, but I dont think it's necessary. It's pretty obvious that if you fall straight down, it has to be before she has moved. I think the useful information here is that there is a fall down strat, which some have not realized

**Comment by osse101** [comment]
👍

**Pull request merged**